js/iframe-coordinator/elements/frame-router.ts

```diff
@@ -1,20 +1,8 @@
-import {
-  Host,
-  HostRouter,
-  ClientRegistrations,
-  LabeledMsg,
-  Publication
-} from "../elm/Host.elm";
+import { Elm, Publication } from "../elm/Host.elm";
 import ClientFrame from "./x-ifc-frame";
-import { Location } from "../libs/types";
 
 const ROUTE_ATTR = "route";
 
-const TOAST_REQUEST_MSG_TYPE = "toastRequest";
-const TOAST_REQUEST_EVENT_TYPE = "toastRequest";
-const NAV_REQUEST_MSG_TYPE = "navRequest";
-const NAV_REQUEST_EVENT_TYPE = "navRequest";
-
```

Comment:
I missed this in the last PR, but rather than having typescript in the loop on message labels and event types, this element just emits a custom DOM event based on the incoming type. This will make message versioning easier to handle, since the knowledge about types and labels is in fewer places.